source/write/delete.txt

```diff
@@ -0,0 +1,180 @@
+.. _ruby-write-delete:
+
+================
+Delete Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: remove, drop, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to remove
+documents from a MongoDB collection by performing **delete operations**.
+
+A delete operation removes one or more documents from a MongoDB collection.
+You can perform a delete operation by using the ``delete_one`` or
+``delete_many`` methods.
```

Comment:
Applies to all instances of these methods
```suggestion
You can perform a delete operation by using the ``delete_one()`` or
``delete_many()`` methods.
```

Reply:
We've been avoiding parentheses in Ruby documentation, as the Ruby API docs don't use them for method names.

Reply:
Yes, I strongly suggest omitting empty parentheses on methods. It's not technically *wrong*, but it looks foreign to Ruby developers. Idiomatically, empty parentheses are always omitted.